packages/contracts/config/graph.sepolia.yml

```diff
@@ -159,3 +159,13 @@ contracts:
       controller: "${{Controller.address}}"
     calls:
       - fn: "syncAllContracts"
+  SubgraphAvailabilityManager:
+    init:
+      governor: "${{Env.deployer}}"
```

Comment:
Is there a reason we don't set the council as the governor from the get go?

Reply:
I was using the deployer while I'm testing so I don't have to use the multisig.